Makefile

```diff
@@ -358,7 +358,7 @@ endef
 define deploy-k8gb-with-helm
 	helm repo add --force-update k8gb https://www.k8gb.io
 	cd chart/k8gb && helm dependency update
-	helm -n k8gb upgrade -i k8gb $5 -f $(VALUES_YAML) \
+	helm -n k8gb upgrade -i k8gb $5 \
```

Comment:
This file is default, when deploying stable, it overwrites values to the new one from PR during upgrade tests

Reply:
Yes, but it will mean that it will ignore local values.yaml modification during `deploy-local-cluster` https://github.com/k8gb-io/k8gb/blob/master/Makefile#L378 . Is it something we want?

Reply:
I personally never used it, and deploy stable with local chart (potentially new) values sounds broken.

Reply:
@k0da I used it frequently for the local chart testing with deploying from scratch. @kuritka @somaritane what about you guys?

Reply:
@ytsarev I can set default value to empty string instead, if that would help

Reply:
@k0da agreed.  i guess that's a nice trade-off